scss/index.scss

```diff
@@ -0,0 +1,134 @@
+@import "./variables.scss";
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+  scroll-behavior: smooth;
+}
+
+.container {
+  position: relative;
+  max-width: 100%;
+  // top: -62px;
+}
+
+.links {
+  position: sticky;
+  top: 0;
+  width: 100%;
+  display: flex;
+  justify-content: space-evenly;
+  padding: 15px 80px;
+  z-index: 1000;
+  @media (max-width: $mediumScreen) {
+    padding: 10px 0px;
+  }
+  background-color: #000;
+  li {
+    transition: all 100ms linear;
+    font-family: $smallerFontStyle;
+    font-size: $smallFont;
+    @media (max-width: $mediumScreen) {
+      font-size: $verySmallFont;
+    }
+    &:hover {
+      transition: all 100ms linear;
+      transform: scale(1.3);
+    }
+    a {
+      color: hsl(300, 62%, 100%);
+      text-decoration: none;
+      font-weight: bold;
+    }
+  }
+}
+
+.section {
+  width: 100%;
+  height: 100vh;
+  background-color: hsl(293, 62%, 100%);
+  display: flex;
+}
+
+.intro {
+  display: grid;
+  min-width: 80%;
+  max-width: 80%;
+  height: 100vh;
+  margin: auto;
+  grid-template-columns: 2fr 1fr;
+  @media (max-width: $mediumScreen) {
+    grid-template-columns: 1fr 0fr;
```

Comment:
nitpick: you can omit `units` from `0` values, it's a small fix but and reduce file size. `#good-coding-habits`